src/main/java/com/google/sps/data/ProcessCareerQuizResults.java

```diff
@@ -0,0 +1,43 @@
+package com.google.sps.data;
+
+import java.util.List;
+import java.util.HashMap; 
+import java.util.Map; 
+import java.util.Map.Entry;
+import com.google.sps.data.CareerQuestionChoice;
+
+/** Returns the recommended career path based off the career quiz results. */
+public class ProcessCareerQuizResults {
+  private ProcessCareerQuizResults() {};
+  
+  public static String getRecommendedCareerPath(List<CareerQuestionChoice> userChoices) {
+    Map<String, Integer> choiceFrequencyHashMap = new HashMap<String, Integer>(); 
+    String careerPath;
+    String maxFrequencyCareerPath = new String();
+    // creates a frequency hashmap mapping associated career paths of choices to their frequencies
+    for (CareerQuestionChoice choice : userChoices) {
+      careerPath = choice.getAssociatedCareerPath();
+      if (choiceFrequencyHashMap.containsKey(careerPath)) {
+        choiceFrequencyHashMap.put(careerPath, choiceFrequencyHashMap.get(careerPath) + 1); 
+      } 
+      else { 
+        choiceFrequencyHashMap.put(careerPath, 1);
+      } 
```

Comment:
This `if/else` block can be reduced with this handy little function:
```
		Integer count = map.getOrDefault(key, 0);
		map.put(key, count + 1);
```

Reply:
very nifty, done.